Formula/ocrmypdf.rb

```diff
@@ -0,0 +1,80 @@
+class Ocrmypdf < Formula
+  include Language::Python::Virtualenv
+
+  desc "Adds an OCR text layer to scanned PDF files"
+  homepage "https://github.com/jbarlow83/OCRmyPDF"
+  url "https://files.pythonhosted.org/packages/9b/c0/13a0ffb2184b85018b64466d1303ec1ceee05acff0b8ada16fdbf2ec101d/ocrmypdf-5.4.4.tar.gz"
+  sha256 "e523591c5d4e4a8cdeee2e9e24c3f631c8638a803dba74c5b8b274681784dbf4"
+
+  depends_on :python3
+  depends_on :x11 # Pillow needs XQuartz
+  depends_on "pkg-config" => :build
+  depends_on "tesseract"
+  depends_on "ghostscript"
+  depends_on "unpaper"
+  depends_on "qpdf"
+
+  # For Pillow source install
+  depends_on "openjpeg"
+  depends_on "freetype"
+  depends_on "libpng"
+  depends_on "jpeg"
+  depends_on "webp"
+  depends_on "little-cms2"
+
+  conflicts_with :cask => "caskroom/cask/mactex", :because => "mactex installs an incompatible ghostscript; use mactex-no-ghostscript instead"
```

Comment:
I don't think this syntax works yet, unless I missed something?

Reply:
It appears `:cask =>` is currently a no-op but "caskroom/cask/mactex" should work (although I could find only one example of it in use). Will change.

Reply:
Yeah, I don't know if this is supported at all yet in `core`, essentially. Paging @ilovezfs for 💡.

Reply:
We don't handle `conflicts_with` casks, so this needs to be removed. Possibly a caveat that brew's ghostscript should be in PATH and not mactex.

Other option: find a way to configure `ocrmypdf` so that it knows to use our ghostscript rather than rely on PATH.

Reply:
> Possibly a caveat that brew's ghostscript should be in PATH and not mactex.

Could check this using a custom `Requirement`, but it'd be pretty gross 🙈.

Reply:
@fxcoudert What do you think of the caveat I added, which amounts to warning if `$(brew --prefix)/bin/gs` is not pointed somewhere in the Cellar? This will be case the if mactex is installed or the user did `brew unlink ghostscript` for some reason.

As an alternative, ocrmypdf supports overriding the path for external executables it calls with environment variables (e.g. `OCRMYPDF_GS=/tmp/ghostscript/gs` will replace Ghostscript). I could use this feature to ensure ocrmypdf uses `/usr/local/opt/<formula>/bin/<binary>` for tesseract, ghostscript, unpaper and qpdf instead of finding them in `PATH`. However this might violate expectations for `PATH`.